pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/SSBQueryIntegrationTest.java

```diff
@@ -0,0 +1,298 @@
+  private void loadSSBToH2()
+      throws Exception {
+    setUpH2Connection();
+    Map<String, String> createTableSqlMap = new HashMap<>();
+    // TODO: refactor these out to .sql files
+    createTableSqlMap.put("dates",
+        "CREATE TABLE dates(D_DATEKEY INT, D_DATE VARCHAR(18), D_DAYOFWEEK VARCHAR(9), D_MONTH VARCHAR(9), "
+            + "D_YEAR INT, D_YEARMONTHNUM INT, D_YEARMONTH VARCHAR(7), D_DAYNUMINWEEK INT, D_DAYNUMINMONTH INT, "
+            + "D_DAYNUMINYEAR INT, D_MONTHNUMINYEAR INT, D_WEEKNUMINYEAR INT, D_SELLINGSEASON VARCHAR(12), "
+            + "D_LASTDAYINWEEKFL INT, D_LASTDAYINMONTHFL INT, D_HOLIDAYFL INT, D_WEEKDAYFL INT)");
+    createTableSqlMap.put("customer",
+        "CREATE TABLE customer(C_CUSTKEY INT, C_NAME VARCHAR(25), C_ADDRESS VARCHAR(25), C_CITY VARCHAR(10), "
+            + "C_NATION VARCHAR(15), C_REGION VARCHAR(12), C_PHONE VARCHAR(15), C_MKTSEGMENT VARCHAR(10))");
+    createTableSqlMap.put("lineorder",
+        "CREATE TABLE lineorder(LO_ORDERKEY INT, LO_LINENUMBER INT, LO_CUSTKEY INT, LO_PARTKEY INT,LO_SUPPKEY INT, "
+            + "LO_ORDERDATE INT, LO_ORDERPRIORITY VARCHAR(15), LO_SHIPPRIORITY VARCHAR(1), LO_QUANTITY INT, "
+            + "LO_EXTENDEDPRICE INT, LO_ORDTOTALPRICE INT, LO_DISCOUNT INT, LO_REVENUE INT, LO_SUPPLYCOST INT, "
+            + "LO_TAX INT, LO_COMMITDATE INT, LO_SHIPMODE VARCHAR(10))");
+    createTableSqlMap.put("part",
+        "CREATE TABLE part(P_PARTKEY INT, P_NAME VARCHAR(22), P_MFGR VARCHAR(6), P_CATEGORY VARCHAR(7), "
+            + "P_BRAND1 VARCHAR(9), P_COLOR VARCHAR(11), P_TYPE VARCHAR(25), P_SIZE INT, P_CONTAINER VARCHAR(10))");
+    createTableSqlMap.put("supplier",
+        "CREATE TABLE supplier(S_SUPPKEY INT,S_NAME VARCHAR(25), S_ADDRESS VARCHAR(25), S_CITY VARCHAR(10), "
+            + "S_NATION VARCHAR(15), S_REGION VARCHAR(12), S_PHONE VARCHAR(15))");
+
+    for (Map.Entry<String, String> tableResource : SSB_QUICKSTART_TABLE_RESOURCES.entrySet()) {
+      try {
+        URL resourceUrl = getClass().getClassLoader().getResource(tableResource.getValue());
+        Assert.assertNotNull(resourceUrl);
+        String tableName = tableResource.getKey();
+        File resourceFile = new File(resourceUrl.getFile());
+        File dataDir = new File(resourceFile.getAbsolutePath(), "rawdata");
+        String createTableSql = createTableSqlMap.get(tableName) + " AS SELECT *  FROM CSVREAD('"
+            + dataDir.getAbsolutePath() + "/" + tableName + ".csv', null, 'fieldSeparator=|');";
+        _h2Connection.prepareCall("DROP TABLE IF EXISTS " + tableName).execute();
+        _h2Connection.prepareCall(createTableSql).execute();
+        LOGGER.warn("load table from " + tableResource + " to H2 successful!");
+      } catch (Throwable t) {
+        throw new RuntimeException("Failed to load table to H2 for: " + tableResource.getKey());
+      }
+    }
+  }
+
+  @DataProvider(name = "ssbQueryDataProvider")
+  public static Object[][] ssbQueryDataProvider() {
```

Comment:
So it seems like this is about testing these SSB queries against the current engine and they are available to try out in the multi-stage via the multi-stage quickstart ?

Do we plan on adding SSB functional test for the multi-stage engine in the form of integration test ? I think they can also be used as a basic test bed in addition to exploring via quick start.

Reply:
Yes. That's the plan. And only v2 multi stage engine can run these queties. 

Sadly I wasn't able to run these test on GHA. For some reason it runs fine for me locally. I will continue to debug a bit

Reply:
lol figured out the issue. it was that when I run local the SSB resources are in file format thus loaded successful. but on GHA it has already been packaged into a JAR thus the way i load it doesn't work. fixing